accounts/lookups.py

```diff
@@ -22,7 +23,7 @@ def get_query(self, q, request, search_ldap=True):
         if (results or search_ldap is False):
             return results
 
-        results = ldap.search_or_create_users(q)
+        results = graph.search_or_create_users(q)
```

Comment:
is there the potential for making this toggleable in config.py? not necessary, just an idea I had

Reply:
i'm sure it could be, all the ldap code is still in place in ldap.py. I would just question the usefulness, however, considering LDAP may be going away in the future, and I'm not even sure whether we currently have any working LDAP access.

Reply:
Fair enough, just a thought I had.